src/arrays.js

```diff
@@ -1,61 +1,80 @@
 const getNthElement = (index, array) => {
-  // your code here
+  const adjustIndex = index % array.length;
+  return array[adjustIndex];
 };
 
 const arrayToCSVString = array => {
-  // your code here
+  return array.join(',');
 };
 
 const csvStringToArray = string => {
-  // your code here
+  return string.split(',');
 };
 
 const addToArray = (element, array) => {
-  // your code here
+  array.push(element);
 };
 
 const addToArray2 = (element, array) => {
-  // your code here
+  const newArray = array.concat(element);
+  return newArray;
 };
 
 const removeNthElement = (index, array) => {
-  // your code here
+  array.splice(index, 1);
 };
 
 const numbersToStrings = numbers => {
-  // your code here
+  const newString = numbers.map(numbers => numbers.toString());
+  return newString;
```

Comment:
in your callback function, you've give the parameter the name `numbers`, which is already used. This won't break your code, but you should use a different name to improve readability, e.g.
```
numbers.map(number => number.toString());
```

Reply:
as above, do not need to assign to variable before returning, i.e. can just do
```
return numbers.map(number => number.toString());
```